app/app.js

```diff
@@ -106,7 +107,7 @@ app.get(['/((\\d+))', '/examples/sample-pub/((\\d+))'], (req, res) => {
   res.render('../app/views/article', {
     swgJsUrl: SWG_JS_URLS[setup.script],
     setup: setup,
-    publicationId: PUBLICATION_ID,
+    publicationId: getPublicationId(),
```

Comment:
Why not just pass the whole config object here?

Reply:
done